readme.txt

```diff
@@ -1,11 +1,11 @@
 # Checkfront Online Booking System
 
 Contributors: checkfront
-Stable tag: 3.7
+Stable tag: 3.8
```

Comment:
I don't think we need to bump this if we didn't actually change anything in the app. This is just what would prompt user to 'Update Checkfront WordPress Plugin'

Reply:
Ok. Thats kinda the only thing that made this testable (since the app description isn't updated until we publish the new version), but I guess thats not a big deal for such a small change